### PR TITLE
govet nil pointer fix

### LINKED ISSUE #116: Govet verification fails with possible nil pointer issue
/kind bug

**What happened?**
Govet verification fails with possible nil pointer issue
```
madhankumar@MADHANKUMARs-MacBook-Pro ibm-powervs-block-csi-driver % make verify
echo "Installing golangci-lint..."
Installing golangci-lint...
curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.43.0
golangci/golangci-lint info checking GitHub for tag 'v1.43.0'
golangci/golangci-lint info found version: 1.43.0 for v1.43.0/darwin/amd64
golangci/golangci-lint info installed ./bin/golangci-lint

### verify-vendor:
Repo uses 'go mod'.
+ env GO111MODULE=on go mod tidy
Go dependencies up-to-date.
echo "verifying and linting files ..."
verifying and linting files ...
./hack/verify-all
Verifying gofmt
No issue found
Verifying govet
Done
pkg/driver/node.go:386:62: SA5011: possible nil pointer dereference (staticcheck)
	klog.V(4).Infof("NodeGetVolumeStats: called with args %+v", *req)
	                                                            ^
pkg/driver/node.go:389:5: SA5011(related information): this check suggests that the pointer can be nil (staticcheck)
	if req == nil || req.VolumeId == "" {
	   ^
make: *** [verify] Error 1
madhankumar@MADHANKUMARs-MacBook-Pro ibm-powervs-block-csi-driver % 
```

**What you expected to happen?**
Don't have to check if req is nil as its passed from provisioner and will not be nil.

**How to reproduce it (as minimally and precisely as possible)?**
Run ` make verify` on the repo.

### PR DESCRIPTION
**Pull request by Madhan-SWE**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Fixes govet verification nil pointer issue

**Which issue(s) this PR fixes**:
Fixes #116

### CODE REVIEW

File: pkg/driver/node.go [hunk 1]
```diff
@@ -386,7 +386,7 @@ func (d *nodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVo
 	klog.V(4).Infof("NodeGetVolumeStats: called with args %+v", *req)
 	var resp *csi.NodeGetVolumeStatsResponse
 
-	if req == nil || req.VolumeId == "" {
+	if req.VolumeId == "" {
```
Comment (mkumatag) [inline pkg/driver/node.go:389]: this still is not resolving the issue, I feel can be written as

```suggestion
	if req != nil && req.VolumeId == "" {
```

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: pkg/driver/node.go (hunk 1 above)
Comment (Madhan-SWE) [reply pkg/driver/node.go]: This doesn't fix the issue. However, I got the right fix.

Comment (Madhan-SWE) [reply pkg/driver/node.go]: Please check the current changes

File: pkg/driver/node.go [hunk 2]
```diff
@@ -383,12 +383,11 @@ func (d *nodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 }
 
 func (d *nodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
-	klog.V(4).Infof("NodeGetVolumeStats: called with args %+v", *req)
 	var resp *csi.NodeGetVolumeStatsResponse
-
 	if req == nil || req.VolumeId == "" {
 		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
 	}
+	klog.V(4).Infof("NodeGetVolumeStats: called with args %+v", *req)
```
Comment (mkumatag) [inline pkg/driver/node.go:390]: But this wont print if volumeID is an empty string, it fails the purpose, may we can add in the beginning only in the beginning inside null check for that req variable.

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
cc @dharaneeshvrd

File: pkg/driver/node.go (hunk 2 above)
Comment (Madhan-SWE) [reply pkg/driver/node.go]: Fixed as suggested

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**